backend/tests/icpy/agent/agents/test_cerebras_gpt_oss_agent_wrapper.py

```
@@ -0,0 +1,32 @@
+Validates agent delegates to GeneralAgent correctly and uses correct model identifiers.
+"""
+from typing import Iterable
+
+from icpy.agent.agents import cerebras_gpt_oss_agent
+from icpy.agent.core.llm.cerebras_client import CerebrasClientAdapter
+
+
+def test_cerebras_gpt_oss_agent_delegates_to_general_agent(monkeypatch):
+    """Test that the Cerebras GPT OSS agent delegates to GeneralAgent correctly."""
+    # Patch adapter.stream_chat to avoid network calls and API keys
+    def fake_stream_chat(self, *, model, messages, tools=None, max_tokens=None) -> Iterable[str]:
```

Ruff (0.14.14)

Unused function argument: `self`

(ARG001)